src/RTCCertificate.cpp

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2016, Andrew Gault and Nick Chadwick.
```

Comment:
Please feel free to append your name to all files, and add 2017 as a copyright year.

I'm not a lawyer, but I feel its easiest to keep all 3 names across all files; that way there's no difference between creating a new file and refactoring in future. That way its also clear that all contributors are releasing their code under the same permissive license.